PerformanceNavigationTiming for User Agent Launch/explainer.md

```diff
@@ -56,65 +64,52 @@ unloadEventStart: 0
+Under normal conditions, if fetchStart (or redirectStart) minus startTime is exceedingly high, one might assume the cause was due to the user agent. Similarly, a large gap between domainLookupEnd and domainLookupStart could suggest DNS issues. However, performance spikes can appear at any point across the navigation timeline if the page is loaded during a contented period making these assumptions invalid.
 
 ![Image](./timestamp-diagram.png)
 
 ## Goals
-The proposed changes in this document aim to enable web application developers to pivot performance telemetry of their web application to eliminate outlier metrics due to user-agent resourcing contention.
+The proposed changes in this document aim to enable web application developers to pivot performance telemetry of their web application to eliminate outlier metrics due to user-agent resource contention.
 
 ## Non-Goals
  * Distinguishing between various implementation-specific startup processes.
  * Signaling browser launches which do not involve a full browser initialization (e.g., restoring a session from a suspended state on mobile platforms).
+ * Identifying specific external factors causing page load contention.
 
 ## Use Cases
  * A web application which typically runs as an installed PWA (progressive web apps) that might get launched without a user agent already running. E.g., PWA’s that run during OS (operating systems) login.
  * A web application which runs as the user's homepage.
 
 ## Proposed Solution
 
-### Add new systemEntropy field to PerformanceNavigationTiming
+### Add new 'confidence' field to PerformanceNavigationTiming
 
-To enable developers to discern if the page load occurs during a non-optimal performance state, we will add a new ‘systemEntropy’ field to the PerformanceNavigationTiming struct. This would be a new enum representing the state of the user agent at the time the navigation was started.
+To enable developers to discern if the navigation timings are representative for their web application, we propose adding a new ‘confidence’ field to the PerformanceNavigationTiming struct.
 
 ```javascript
-enum NavigationEntropy {
+enum NavigationConfidence {
     "high",
-    "normal"
+    "low"
 };
 ```
-The payload of a performance.getEntriesByType("navigation") call would look like:
+
+The payload of a performance.getEntriesByType("navigation") call would then look like:
 
 ```javascript
+confidence: "high"
 connectEnd: 126.19999998807907
 connectStart: 126.19999998807907
 <snip>
-serverTiming: []
-startTime: 0
-systemEntropy: "high"
-transferSize: 300
-type: "navigate"
-unloadEventEnd: 0
 unloadEventStart: 0
 ```
-An example of how a web application might use this:
 
-```javascript
```

Comment:
I think it would be worth having a few sentences to explain how we expect site owners to aggregate and reason about the performance data given the `confidence` signal with injected randomness.